From 38fed293117a10bf3cc553801313500d7b2d5332 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 21 Feb 2022 09:26:18 +0200
Subject: [PATCH] [SW-0] gaudi3: fix compilation

This patch adds changes to fix compilation of arcboot for gaudi3
target.

Change-Id: I3291ae7b6da45c886d2cbf2d780fb915758b53a6
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 6475a59..692dd37 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -267,16 +267,17 @@
 	err = dma_copy_image_v2(desc->img_size, desc->spi_offset,
 				desc->loadaddr, DMA_DEST_HBW, false);
 	if (err != DMA_SUCCESS) {
-		if (err == DMA_INVALID_CRC_ERROR)
+		if (err == DMA_INVALID_CRC_ERROR) {
 			security_report_img_status(cur_img, IMG_STATUS_CRC_FAIL);
 			arcboot_rom_status_update_bringup(img_sts_base,
 					PROGRESS_BRINGUP_CRC_FAIL);
-		else if (err == DMA_UNALIGNED_DEST_ADDR)
+		} else if (err == DMA_UNALIGNED_DEST_ADDR) {
 			arcboot_rom_status_update_bringup(img_sts_base,
 					PROGRESS_BRINGUP_UNALIGNED_DEST_ADDR);
-		else if (err == DMA_UNALIGNED_IMG_SIZE)
+		} else if (err == DMA_UNALIGNED_IMG_SIZE) {
 			arcboot_rom_status_update_bringup(img_sts_base,
 					PROGRESS_BRINGUP_UNALIGNED_IMG_SIZE);
+		}
 
 		return ARCBOOT_BRINGUP_ERROR;
 	}
@@ -304,13 +305,13 @@
 			pub_key, desc->pub_key_size, signature, desc->sig_size, desc->sec_ver);
 	if (ret != SECUREBOOT_VERIFY_SUCCESS) {
 		if (ret == SECUREBOOT_VERIFY_VER_FAILED)
-			security_report_img_status(curr_img, IMG_STATUS_FW_VERSION_LOW);
+			security_report_img_status(cur_img, IMG_STATUS_FW_VERSION_LOW);
 		else if (ret == SECUREBOOT_VERIFY_KEY_REVOKED)
 			arcboot_rom_status_update_bringup(img_sts_base, PROGRESS_BRINGUP_KEY_REVOKED);
 		else if (ret == SECUREBOOT_VERIFY_KEY_FAILED)
 			arcboot_rom_status_update_bringup(img_sts_base, PROGRESS_BRINGUP_KEY_AUTH_FAIL);
 		else
-			security_report_img_status(curr_img, IMG_STATUS_SIG_AUTH_FAIL);
+			security_report_img_status(cur_img, IMG_STATUS_SIG_AUTH_FAIL);
 		return ARCBOOT_BRINGUP_ERROR;
 	}
 
